File: test_data/code/python/1859-sorting-the-sentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        words = ["" for _ in range(9)]
        temp = [i for i in s.split(' ')]
        for word in temp:
            for char in word:
                if char.isdigit():
                    index = int(char)
                    words[index - 1] = (words[index - 1] + word.replace(str(index), '') + ' ')
                    break
        # Remove trailing space, join and return
        return ' '.join(map(lambda x: x.strip(), words)).strip()

File: test_data/code/java/1859-sorting-the-sentence.java
class Solution {
    public String sortSentence(String s) {
        String[] words = s.split(" ");
        StringBuilder sb = new StringBuilder();
        Arrays.sort(words, (a, b) -> Integer.parseInt(a.substring(a.length() - 1)) - Integer.parseInt(b.substring(b.length() - 1)));
        for (String word : words) {
            sb.append(word.substring(0, word.length() - 1)).append(" ");
        }
        return sb.toString().trim();
    }
}